modules/ossm-tutorial-bookinfo-install.adoc

```diff
@@ -31,8 +36,6 @@ The commands in this section assume the {SMProductShortName} control plane proje
 
 .Procedure
 
-. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
-
```

Comment:
**PEER REVIEWER** I removed the log in step and made it a prereq to align with other modules in the book.